seed/migrations/0125_dq_refactor.py

```diff
@@ -0,0 +1,62 @@
+# Generated by Django 2.2.10 on 2020-04-13 23:03
+
+from django.db import migrations, models
+
+
+def forwards(apps, schema_editor):
+    Rule = apps.get_model('seed', 'Rule')  # Total 1226
+    # =========================================================================================================== #
+    # data_type not null ---> 1 ---> text_match null: not_null/require/''                                         #
+    #                    |      |--> test_match not null ---> text_match != '': include                           #
+    #                    |                               |--> text_match == '': not_null/required/''              #
+    #                    |                                                                                        #
+    #                    |--> != 1 ---> min & max null: not_null/required/''                                      #
+    #                              |--> min / max not null: range                                                 #
+    #                                                                                                             #
+    # data_type null -------> min & max null ---> text_match null: not_null/required/''                           #
+    #                |                       |--> text_match not null ---> text_match == '': not_null/required/'' #
+    #                |                                                |--> text_match != '': include              #
+    #                |--> min / max not null ---> range                                                           #
+    # =========================================================================================================== #
+    Rule.objects.filter(data_type=1, text_match=None, not_null=True).update(condition='not_null')
+    Rule.objects.filter(data_type=1, text_match=None, required=True, not_null=False).update(condition='required')
+    Rule.objects.filter(data_type=1, text_match=None, required=False, not_null=False).update(condition='')
```

Comment:
Currently this condition does not check anything. And yes my intent for leaving the condition blank is: 1. to alert and prompt users that no condition has been selected for this rule and there should be one; 2. to prompt users to hopefully clean up rule group since There are rules in the db where there's only data type and nothing else.